assets/js/feature-tours/pagespeed-insights-force-active.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Page Speed Insight Force Active tour for the dashboard.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import { VIEW_CONTEXT_DASHBOARD } from '../googlesitekit/constants';
+import {
+	MODULES_PAGESPEED_INSIGHTS,
+	STRATEGY_DESKTOP,
+	STRATEGY_MOBILE,
+} from '../modules/pagespeed-insights/datastore/constants';
+import { CORE_SITE } from '../googlesitekit/datastore/site/constants';
+
+const pagespeedInsightsForceActive = {
+	slug: 'pagespeedInsightsForceActive',
+	contexts: [ VIEW_CONTEXT_DASHBOARD ],
+	version: '1.45.0',
```

Comment:
I think this should be `1.46.0` since we are going to display this feature tour only for users who have installed the plugin earlier.

Reply:
https://github.com/google/site-kit-wp/pull/4305/commits/e127eb1e08d6fc09cfb94ff99bde0f7761f56520

Reply:
@ivankruchkoff @eugene-manuilov Sounds good - we just need to ensure then that this actually ships with 1.46.0. I've therefore set this release on the issue to mark it already as "requirement".

Overall, where does the semver error come from? It would be great if we could specify `n.e.x.t` here as we do elsewhere - maybe we can allow for a special case (e.g. `n.e.x.t` could _always_ consider be greater than any other version you compare it too). Nothing to break our heads about here, but maybe worth a separate issue.